pages/playerprofile.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState } from "react";
+
+const PlayerNavBar: React.FunctionComponent = () => {
+  const [title, setTitle] = useState("Overview");
+  const Button = (category: string): unknown => {
+    return (
+      <button
+        type="button"
+        className={
+          title === category
+            ? "bg-gray-300 py-3 px-8 rounded-full"
+            : "py-3 px-8 rounded-full"
+        }
+        onClick={() => {
+          setTitle(category);
+        }}
+      >
+        {category}
+      </button>
+    );
+  };
+  return (
+    <div>
+      <div className="flex flex-row justify-between text-xs text-center">
+        {Button("Overview")}
+        {Button("Engagement")}
+        {Button("Academic Performance")}
+        {Button("Attendance")}
+        {Button("Physical Wellness")}
+        {Button("Highlights")}
+      </div>
+      <div className="mt-12 mb-10 text-xl">{title}</div>
```

Comment:
can you add a comment noting a TODO to replace this with the profile content in the next sprint for future reference?